tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -216,6 +240,47 @@ var activationResultCmd = &cobra.Command{
     },
 }
 
+// lastFlag(args) retrieves the last activation with flag -l or --last
+// Param: Brings in []strings from args
+// Return: Returns a []string with the latest ID or the original args and any errors
+func lastFlag(args []string) ([]string, error) {
+  // Checks to see if there is an ID sent with the --last
+  // If an ID is given with --last then an error will be thrown
+  if  flags.activation.last {
+    options := &whisk.ActivationListOptions {
+      Limit: 1,
+      Skip: 0,
+    }
+    activations,_, err := client.Activations.List(options)
+    if err != nil {   // Checks Activations.List for errors when retrieving latest activaiton
+      whisk.Debug(whisk.DbgError, "client.Activations.List() error for flag --last: %s\n", err)
+      return args, err
+    }
+    if len(activations) == 0 {    // Checks to to see if there are activations available
+      whisk.Debug(whisk.DbgError, "No activations found in activation list\n")
+      errStr := wski18n.T("Activation list contains no activations")
+      whiskErr := whisk.MakeWskError(errors.New(errStr), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+      return args, whiskErr
+    } else {
+      if len(args) == 0 {
+        whisk.Debug(whisk.DbgInfo, "Appending most recent activation ID into args\n")
```

Comment:
Should probably print the arguments you are appending in the debug messages in this function.